session/pingpong/invoice_tracker.go

```diff
@@ -19,9 +19,9 @@ package pingpong
 
 import (
 	"bytes"
-	"crypto/rand"
 	"encoding/hex"
 	"math"
+	"math/rand"
```

Comment:
Is it ok to use "math/rand" instead of "crypto/rand" here? Do we need to protect this ID somehow?

Reply:
I think it's good enough, we don't really need to be super duper secure here.

Reply:
Besides, it's in the providers interest to not to tamper with this, if he does - well, it's his own responsibility :)